api/services/Flairs.js

```diff
@@ -261,6 +263,13 @@ exports.makeNewCSSClass = function (previous_flair, new_addition, subreddit) {
   if (new_addition === 'involvement') {
     return previous_flair.replace(/( |$)/, '1$1');
   }
+  if (extraFlair.includes(new_addition)) {
```

Comment:
If we're using `Array#includes`, we can no longer support Node 4. I'm fine with this, but we might want to update the readme.